Add migration for isMaster column to isSchedulingPublisher in UmbracoServer table

Jeavon opened the pull request:
Following on from https://github.com/umbraco/Umbraco-CMS/pull/10480 In this PR I added a migration for isMaster column to isSchedulingPublisher in UmbracoServer table

Creating this as a draft PR for now as my local v9 is having problems so I've not yet verified it works

bergmania commented:
Could potentially fail with SqlCE.. But needs to be tested for both SqlServer and SqlCE

Jeavon commented:
Fixed my build (somehow I had a hooky version of Examine 2.0 that had different method signatures in my package cache! 🤯)

Tested with CE 
Before
![image](https://user-images.githubusercontent.com/1829920/134389655-92c62081-deb0-45e3-8494-d2f9c2184049.png)
After
![image](https://user-images.githubusercontent.com/1829920/134389700-766ea33c-9632-4d54-91f9-bbbbed7cf2dd.png)

And tested with LocalDB

All good.

I did find 2 isMaster references in SQL that I've updated in https://github.com/umbraco/Umbraco-CMS/pull/11167/commits/a3ca09c82e690859c132488eb68290237dfd0643 which have updated this PR
